addons/radar/XEH_preInit.sqf

```diff
@@ -5,6 +5,10 @@ ADDON = false;
 #include "\a3\ui_f\hpp\defineDIKCodes.inc"
 #define CBA_SETTINGS_CAT (format ["%1 - %2",localize "STR_dui_mod", localize "STR_dui_addon_radar"])
 
+private _res = getResolution;
+private _height = _res select 1;
+private _saneScale = _height / 1080; // diwako calibrates for 1080p so let's do the same
```

Comment:
cyruz suggests `s/_height/(getResolution #1)/` and explaining that we scale by height in the comment

Reply:
I see it like that as well, there is not really a need for 3 variables, one will do fine and if it is explained for what it is used, then it is more than adequate.